scrapers/Andomark.yml

```diff
@@ -0,0 +1,34 @@
+# Be sure to set a User Agent in Settings! (ex:Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:79.0) Gecko/20100101 Firefox/79.0)
+name: "Andomark"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - reidmylips.elxcomplete.com/updates/
+      - sheseducedme.com/updates
+      - aliciasgoddesses.com/updates
+      - ariellynn.com/tour/
+      - ravenswallowzxxx.com/updates/
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //span[@class="update_title"]/text()|//title/text()
+      Date:
+        selector: //span[@class="availdate"]/text()[1]|//span[@class="update_date"]/text()
+        postProcess:
+            - parseDate: 01/02/2006
+      Details: //span[@class="latest_update_description"]/text()
```

Comment:
details doesn't work in some scenes when there a url in the description (eg https://ariellynn.com/tour/updates/Girls-Night.html)
leaving out the `text()` does the job
```yaml
      Details: //span[@class="latest_update_description"]
```